doc/whats_new/v0.21.rst

```diff
@@ -391,3 +391,10 @@ These changes mostly affect library developers.
   when `fit` is called twice with the same data, the ouput of
   `predict`, `predict_proba`, `transform`, and `decision_function` does not
   change. :issue:`12328` by :user:`Nicolas Hug <NicolasHug>`
+
+- Add ``test_check_class_weight_balanced_linear_classifier`` to                 
```

Comment:
I would simplify a bit the message:

```
Add an additional common test for estimators to check that ...
```

Reply:
What do you think abou this version:

>  - Add an additional estimator_check test to verify the proper functioning of the check that corroborates that estimators generate correct balanced class weights (when ``class_weight=balanced``). :issue:`9079`  by :user:`Pavel Soriano <psorianom>`.

As I am adding a test for a check, it gets a little bit confusing :/

Reply:
oh upsy. Sorry about that. You actually don't need an entry then. Let me change this and merge then